docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md

```diff
@@ -27,7 +27,7 @@ type: docs
   </li>
 </ul>
 
-YugabyteDB Anywhere installations include configuration settings, certificates and keys, and other components required for creating and managing YugabyteDB universes.
+<br>YugabyteDB Anywhere installations include configuration settings, certificates and keys, and other components required for creating and managing YugabyteDB universes.
```

Comment:
I'm not sure how the style sheet can handle this particular case. We need more vertical space after tabs when they are followed by a para (not header).
Vertical spacing in general needs to be improved. There are a lot of cases when it's not handled well because it doesn't address every single case (nested lists, images within different list levels, paras within lists, etc). And there's way too much space before headers right now.